src/style.css

```
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
```

Шрифты наследуются, большинством элементов, а для input надо прописать явно (но не *)
```suggestion
html {
```